cmd/core-command/Attribution.txt

```diff
@@ -154,6 +154,9 @@ https://github.com/golang/net/blob/master/LICENSE
 gopkg.in/yaml.v2 (Apache 2.0) https://github.com/go-yaml/yaml/
 https://github.com/go-yaml/yaml/blob/v2.2.2/LICENSE
 
+gopkg.in/yaml.v3 (MIT) https://github.com/go-yaml/yaml/
```

Comment:
I don't see we are using yaml.v3 in go.mod.  Why do you add this section?

Reply:
It is brought in by testify

Reply:
do we have to upgrade testify?  or, should we use yaml.v3 and remove yaml.v2 in core-metadata later?

Reply:
testify 1.6.1 is not required, just keeping updated with latest. Note yaml.v3 use in testify doesn't impact our use of yaml.v2 in core-metadata. We should consider upgrading to yaml.v3 in core-metadata as a separate issue.